### DISCUSSION TITLE
'Ask a Question' discussion template does not make sense in all cases

### DISCUSSION THREAD

**Discussion by WilliamDEdwards** [renovatebot/renovate#23618] [Request Help]
### How are you running Renovate?

N/A

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

N/A

### Describe the problem

In my opinion, the 'Ask a Question' discussion template does not make sense in all cases. For example, the field "What you tried so far." is required, but not all questions require the user to try anything. The field "Was this something which used to work for you, and then stopped?" is also rarely applicable - there is no indication that anything would or would not have worked or work.

### Relevant debug logs

N/A

### Have you created a minimal reproduction repository?

N/A

**Comment by HonkingGoose** [comment]
The goal of the "Ask a Question" Discussion form is to make it easy for the maintainers to try to answer the question. This also means we want the user to make _some_ effort to give us the right information up front. We sometimes had a back-and-forth with users to get the information. That's time-consuming and annoying for the maintainers.

We ask **Was this something which used to work for you, and then stopped?** to discover regression bugs. Maybe we accidentally broke something that used to work before. Most of the time, it never worked for a user in the first place, because of bad configuration, or wrong expectations.

We ask **Wanted end result.** to learn what the user wants Renovate to do. Sometimes users are looking in the wrong place, or are configuring Renovate in a odd way. So we want to know their desired end-state.

**What you tried so far.** is to learn what the user tried already, or what they read in the documentation.
